misc/profiles2/lookups.dat

```diff
@@ -30,34 +30,30 @@ highway;0000039003 platform
 highway;0000037192 proposed planned virtual
 highway;0000010307 raceway
 highway;0000003152 rest_area
-highway;0000002942 abandoned disused razed demolished dismantled
+highway;0000002942 abandoned disused razed demolished dismantled no
 highway;0000002631 services
 highway;0000002133 corridor
-highway;0000002093 crossing
 highway;0000001440 bus_stop
-highway;0000001274 yes
-highway;0000000679 unsurfaced
-highway;0000000108 byway
-highway;0000000037 driveway
-highway;0000000021 mini_roundabout
```

Comment:
The [`highway=mini_roundabout`](https://wiki.openstreetmap.org/wiki/Tag:highway%3Dmini_roundabout#Mini-roundabouts) tag is used in cases where the island of a traffic circle is traversable. If it is removed, such traffic circles can no longer be detected, because there is no road with [`junction=roundabout`](https://wiki.openstreetmap.org/wiki/Tag:junction%3Droundabout#Roundabouts) there either.

Reply:
Same comment as above, this is in way context.